packages/web/src/scss/components/Button/README.md

```diff
@@ -23,3 +23,12 @@ Disabling a Button:
 <button type="button" class="Button Button--primary is-disabled">Disabled Button</button>
 <button type="button" class="Button Button--primary Button--disabled">Disabled Button</button>
 ```
+
+Sizes:
+
+```html
+<button type="button" class="Button Button--primary Button--medium">Medium Button</button>
+<button type="button" class="Button Button--primary Button--inverted Button--medium">Medium Inverted Button</button>
```

Comment:
Primary and inverted variants are mutually exclusive—it's either one, or the other.